packages/synapse-react-client/src/components/GenericCard/GenericCardActionButton.tsx

```diff
@@ -2,11 +2,12 @@ import { Button } from '@mui/material'
 import { styled } from '@mui/system'
 
 const GenericCardActionButton = styled(Button)({
-  display: 'flex',
   height: '20px',
   padding: '2px 8px',
   borderColor: 'grey.400',
+  fontWeight: 700,
+  lineHeight: '20px',
   '& .MuiButton-startIcon': { marginRight: '4px', marginLeft: 0 },
-})
+}) as typeof Button
```

Comment:
Fixes type issue when using styled button as a link (passing `href` and `target` as props)